crypto/params.c

```diff
@@ -518,7 +535,14 @@ int OSSL_PARAM_get_uint64(const OSSL_PARAM *p, uint64_t *val)
         switch (p->data_size) {
         case sizeof(double):
             d = *(const double *)p->data;
-            if (d >= 0 && d <= INT64_MAX && d == (uint64_t)d) {
+            if (d >= 0
+                    /*
+		     * By subtracting 65535 (2^16-1) we cancel the low order
```

Comment:
There seems to be something wrong with the amount of spaces

Reply:
Some tabs crept in somehow.  Fixed now.